x/upgrade/client/cli/parse.go

```diff
@@ -0,0 +1,73 @@
+package cli
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	gov "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/spf13/cobra"
+)
+
+// softwareUpgradeProposal defines a software upgrade proposal.
+type softwareUpgradeProposal struct {
+	Plan     json.RawMessage
+	Metadata []byte
+	Deposit  string
+}
+
+func parseSubmitSoftwareUpgradeProposal(cdc codec.Codec, path string) (*types.Plan, []byte, sdk.Coins, error) {
```

Comment:
Is this function used somewhere? I thought we were not doing CLI for software upgrade in this PR

Reply:
Doesn't seem to be. I've removed it